gnu.elf/plugin.xml

```diff
@@ -2939,6 +3167,42 @@
                id="arc.gnu.elf.toolchain.hs">
          </toolChain>
       </template>
+      <template
```

Comment:
Upon testing I am now not sure we need those templates at all - HS6x toolchain is already assigned to the "Default" hello world and empty projects, so you don't need duplicate templates.

![image](https://user-images.githubusercontent.com/602123/141497223-ac0d4d50-76c0-4bc8-911a-515ff2171a24.png)